openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1720,11 +1720,6 @@ The following is a non-normative example of an object comprising `credentials_su
 
 #### Credential Offer
 
-The following additional claims are defined for this Credential format to be added to the object entry of the `credentials` parameter defined in (#credential_offer_parameters).
-
-* `credential_definition`: REQUIRED. Object containing the detailed description of the credential type. It consists at least of the following sub claims:
-  * `type`: REQUIRED. Array as defined in (#server_metadata_jwt_vc_json). This claim contains the type values the Wallet shall request in the subsequent Credential Request.
-
 The following is a non-normative example of an object comprising `credentials_supported` parameter of Credential format `jwt_vc_json`:
```

Comment:
It's not a new problem but I think this sentence is incorrect?

```suggestion
The following is a non-normative example of a Credential Offer of type `jwt_vc_json`:
```

Reply:
`jwt_vc_json` is not credential type, it is credential format.

Reply:
Okay, that's a good point and I think that means some of the others are wrong then? But for starters would this be right:

```suggestion
The following is a non-normative example of a Credential Offer for Credential format `jwt_vc_json`:
```

That example is definitely not an example of `credentials_supported` so the current sentence is definitely wrong.

Reply:
I think you are correct, good catch

Reply:
resolved